docs/_sass/_landingPage.scss

```diff
@@ -631,13 +631,6 @@ body {
     color: $home-page-action-color;
 }
 
-.sap-icon--decline::before {
-    font-size: 20px;
-    transition: opacity .3s, transform .3s;
-    transform: rotate(-180deg);
-    color: $home-page-action-color;
-}
-
```

Comment:
Why do you remove this? It's part of the landing page

Reply:
It was causing issues with the decline button on the shellbar search input, and I could not find where this was actually being used in the docs so I thought it might've been leftover... where would this icon be found on the landing page?  I'm not seeing it